### PR TITLE
[NEWS] add v0.6 release

### PR DESCRIPTION
**Pull request by yzhliu**
@tqchen

### CODE REVIEW

File: NEWS.md [hunk 1]
```diff
@@ -26,6 +26,899 @@ Refer to the Roadmap issue for complete list on on-going version features.
 If you check in something that is not reflected in Roadmap issue, please reply
 to that issue so it can get added.
 
+## 0.6
+
+### Relay in Production
+Relay is a functional, differentiable programming language designed to be an expressive intermediate representation for machine learning systems. Relay supports algebraic data types, closures, control flow, and recursion, allowing it to directly represent more complex models than computation graph-based IRs (e.g., NNVM) can. In TVM v0.6, Relay is in stable phase and is ready for production.
+
+* Algebraic Data Types (ADT) support ([#2442](https://github.com/apache/incubator-tvm/pull/2442), [#2575](https://github.com/apache/incubator-tvm/pull/2575)). ADT provides an expressive, efficient, and safe way to realize recursive computation (e.g., RNN). Refer to https://docs.tvm.ai/langref/relay_adt.html for more information.
```
Comment (tqchen) [inline NEWS.md]: can we remove the link to hashtags (e.g. #2242 )to simplify the content?

Comment (yzhliu) [reply NEWS.md]: @tqchen updated. please check again.

File: NEWS.md [hunk 2]
```diff
@@ -26,6 +26,899 @@ Refer to the Roadmap issue for complete list on on-going version features.
 If you check in something that is not reflected in Roadmap issue, please reply
 to that issue so it can get added.
 
+## 0.6
+
+### Relay in Production
+Relay is a functional, differentiable programming language designed to be an expressive intermediate representation for machine learning systems. Relay supports algebraic data types, closures, control flow, and recursion, allowing it to directly represent more complex models than computation graph-based IRs (e.g., NNVM) can. In TVM v0.6, Relay is in stable phase and is ready for production.
+
+* Algebraic Data Types (ADT) support ([#2442](#2442), [#2575](#2575)). ADT provides an expressive, efficient, and safe way to realize recursive computation (e.g., RNN). Refer to https://docs.tvm.ai/langref/relay_adt.html for more information.
```
Comment (tqchen) [inline NEWS.md]: remove the bracket [#2442](#2442) because it will only link to a internal location of news which

Comment (yzhliu) [reply NEWS.md]: you mean sth like `Algebraic Data Types (ADT) support (#2442, #2575)` ?

Comment (tqchen) [reply NEWS.md]: yes

Comment (tqchen) [reply NEWS.md]: ping @yzhliu

**Pull request merged**